src/components/Signin.jsx

```diff
@@ -0,0 +1,282 @@
+import React,{useState} from 'react';
+import undraw_login from '../components/assets/images/undraw_Login.svg';
+import styled from 'styled-components';
+
+const SocialLinks = styled.ul`
+  display: inline;
+  list-style:none;
+`;
+
+const Content = styled.div`
+  width: 70%;
+`;
+const Authenticate =styled.button`
```

Comment:
create a single button for forget password and signIn. Use CSS flex to implement.
ButtonWrapper -
 - Button